cmd/kubeadm/app/util/apiclient/idempotency.go

```diff
@@ -285,6 +285,24 @@ func PatchNodeOnce(client clientset.Interface, nodeName string, patchFn func(*v1
 	}
 }
 
+// CreateNode creates a Node with the given name if the Node doesn't exist.
+// Adding a placeholder v1.LabelHostname label makes the object suitable for patching using PatchNodeOnce.
+func CreateNode(client clientset.Interface, nodeName string) error {
+	node := &v1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   nodeName,
+			Labels: map[string]string{v1.LabelHostname: ""},
```

Comment:
Do we need to pre-fill `LabelHostname`? Kubelet is going to fill in the correct value itself.
If we need to fill it for some reason upon create, we should probably comment here why and supply it with `nodeName` as a value.

Reply:
the comment on top of the function says:
```
Adding a placeholder v1.LabelHostname label makes the object suitable for patching using PatchNodeOnce.
```
prefiling it with nodeName is fine too.
frankly i don't see the need for PatchNodeOnce to even require it.

Reply:
i'm leaving this out. the comment above function definition now has more info.